### DISCUSSION TITLE
Sui Move项目发起：Sui Defi生态收益聚合器

### DISCUSSION THREAD

**Discussion by zengxinhai** [NonceGeek/Move-Camp#228] [Amazing Things!]
收益聚合器的灵感来自以太坊上的[Yearn](https://yearn.finance/vaults), 核心功能就是将分散于各个项目的质押池收益做一个聚合。用户可以一站式的了解自己手上的币有哪些潜在的收益机会。
但我们既然是在Sui上做项目，可以利用Sui本身的优势，进一步实现更强大的功能。

1. 封装一个底层的质押协议，简化项目方开发质押池的同时，大幅提高互操作性
2. 支持无许可添加收益池，无需任何DAO的介入

项目的工作分为3块：
1. 质押协议的合约层开发
2. 配套的TS SDK
3. 前端交互UI

现阶段2 - 3人的团队较为合适，需要有前端经验或Sui Move经验的队友，预计每天需要花费4个小时时间做开发。

**Comment by ChaoRenYuan** [comment]
已加入🙋‍♂️

**Comment by xiegudong45** [comment]
你好，想问下现在还有机会加入吗？
